tex/latex/pgf/doc/pgfmanual-en-macros.tex

```diff
@@ -1727,6 +1728,7 @@
           {%
             \returntospace%
             \commenthandler%
+            \obeylines%
```

Comment:
Do we really need `\obeylines` in the presence of `doc2`?

Reply:
That's fair, but please add a comment to explain the purpose. Then you can merge.